.github/workflows/push.yml

```
@@ -0,0 +1,81 @@
+name: Push
+on:
+  push:
+    paths-ignore:
+      - docs/*
+      - NEWS
+      - UPGRADING
+      - UPGRADING.INTERNALS
+      - README.md
+      - CONTRIBUTING.md
+      - CODING_STANDARDS.md
+    branches:
+      - PHP-7.4
+      - PHP-8.0
+      - PHP-8.1
+      - master
+  pull_request:
+    branches:
+      - '**'
+jobs:
+  LINUX_X64:
+    strategy:
+      matrix:
+        include:
+          - debug: true
+            zts: false
+          - debug: false
+            zts: true
+    name: "LINUX_X64_${{ matrix.debug && 'DEBUG' || 'RELEASE' }}_${{ matrix.zts && 'ZTS' || 'NTS' }}"
+    runs-on: ubuntu-20.04
+    steps:
+      - name: git checkout
+        uses: actions/checkout@v2
+      - name: Create mssql container
+        uses: ./.github/actions/mssql
+      - name: apt
+        uses: ./.github/actions/apt-x64
+      - name: ./configure
+        uses: ./.github/actions/configure-x64
+        with:
+          configurationParameters: >-
+            --${{ matrix.debug && 'enable' || 'disable' }}-debug
+            --${{ matrix.zts && 'enable' || 'disable' }}-zts
+      - name: make
+        run: make -j$(/usr/bin/nproc) >/dev/null
```

```suggestion
        run: make -j$(/usr/bin/nproc)
```
The output is automatically collapsed by GitHub Actions. Having the output might be useful in error cases when needing to debug something.

---

I'm not super convinced the output is useful here, since everything important goes to `stderr`. I'll leave it like this for now but we can always adjust it if necessary :slightly_smiling_face: Thanks for the review!

---

That's fair, you're the one(s) who need to work with this the most, so use whatever works for you.

I've worked extensively on HAProxy's CI, so my suggestions come from my experience with that.

While I'm at it, as it's very similar to the suggestion here: Do you know about GitHub Action's feature to group the output? https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#grouping-log-lines

For HAProxy we use that to collapse the `ldd` output in the “Show HAProxy Version” step: https://github.com/haproxy/haproxy/runs/5300801977?check_suite_focus=true. Click “Show HAProxy Version”, then “Show dynamic libraries.”. It's very useful to have output that is not needed most of the time, but can be helpful to debug issues.

---

Yes, I've tried it to get rid of redundancy between job steps but sadly it didn't work across actions, but I have only tried quickly.